From 766c4bb7bdc0dba89a5c01686f60fe16e04d00fb Mon Sep 17 00:00:00 2001
From: Mark Brown <broonie@linaro.org>
Date: Thu, 8 Aug 2013 21:23:19 +0100
Subject: [PATCH] configs: Remove EDID firmware from Arndale config

It has not been backported, we will have to take a decision about
backporting any extra support and what will be included.

Signed-off-by: Mark Brown <broonie@linaro.org>
---
 linaro/configs/arndale.conf | 1 -
 1 file changed, 1 deletion(-)

diff --git a/linaro/configs/arndale.conf b/linaro/configs/arndale.conf
index ee23bcc..90ce9b7 100644
--- a/linaro/configs/arndale.conf
+++ b/linaro/configs/arndale.conf
@@ -18,7 +18,6 @@ CONFIG_CPU_FREQ_GOV_USERSPACE=y
 CONFIG_VFP=y
 CONFIG_NEON=y
 CONFIG_PM_RUNTIME=y
-CONFIG_EXTRA_FIRMWARE="edid-1920x1080.fw"
 CONFIG_BLK_DEV_LOOP=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_CHR_DEV_SG=y
-- 
1.8.0

